ert3/stats/_stats.py

```diff
@@ -28,6 +30,18 @@ def sample(self):
         else:
             return {idx: float(val) for idx, val in zip(self._index, sample)}
 
+    def ppf(self, x):
+        x = np.full(self._size, x)
+        result = scipy.stats.norm.ppf(x, loc=self._mean, scale=self._std)
+        if self._index is None:
+            return result
+        else:
+            return {idx: float(val) for idx, val in zip(self._index, result)}
```

Comment:
Candidate for function in a base class?

Reply:
Yeah, I was actually planning to write it up as an issue to create a base class representing what we expect from a distribution... I can also address it in this PR if you want me to :)

Reply:
I think leaving it in a separate issue is fine!

Reply:
Made an issue https://github.com/equinor/ert/issues/1357